WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFragment.kt

```diff
@@ -68,11 +68,13 @@ class PostListFragment : Fragment() {
     private val postViewHolderConfig: PostViewHolderConfig by lazy {
         val displayWidth = DisplayUtils.getDisplayPixelWidth(context)
         val contentSpacing = nonNullActivity.resources.getDimensionPixelSize(R.dimen.content_margin)
+        postListType = requireNotNull(arguments).getSerializable(EXTRA_POST_LIST_TYPE) as PostListType
         PostViewHolderConfig(
                 photonWidth = displayWidth - contentSpacing * 2,
                 photonHeight = nonNullActivity.resources.getDimensionPixelSize(R.dimen.reader_featured_image_height),
                 isPhotonCapable = SiteUtils.isPhotonCapable(site),
-                imageManager = imageManager
+                imageManager = imageManager,
+                postListType = postListType
```

Comment:
I believe we don't use the PostViewHolderConfig .postListType anymore.

Reply:
Fixed in 9e445944ef873d880962c0f77dc2079ec0e39ed6